config/custom.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/mitchellh/mapstructure"
 	"google.golang.org/protobuf/encoding/protojson"
 	"google.golang.org/protobuf/proto"
-	"gopkg.in/yaml.v2"
+	"gopkg.in/yaml.v3"
```

Comment:
@travisgroth thoughts? https://github.com/go-yaml/yaml/tree/v3#compatibility

Reply:
That seems okay to me